compiler/rustc_builtin_macros/src/test.rs

```diff
@@ -207,30 +207,30 @@ pub(crate) fn expand_test_or_bench(
     };
 
     let test_fn = if is_bench {
-        // A simple ident for a lambda
-        let b = Ident::from_str_and_span("b", attr_sp);
-
+        // A collision-resistant ident for a lambda, using the user's function name within it to avoid collisions.
```

Comment:
this is now repeating "collision", and should probably also link to the bug this solves.
```suggestion
        // try to avoid name collisions, see bug #148275
```